scans/requirements.txt

```diff
@@ -0,0 +1,4 @@
+urllib3<1.25,>=1.21.1
+zapcli
+pyotp
+selenium
```

Comment:
How would you feel about moving this to something like `securedrop/requirements/python3/scan-requirements.txt` (with or without a pre-processed `scan-requirements.in`)?